plugins/woocommerce/includes/class-wc-validation.php

```diff
@@ -70,6 +70,7 @@ public static function is_postcode( $postcode, $country ) {
 				$valid = (bool) preg_match( '/^(DK-)?([1-24-9]\d{3}|3[0-8]\d{2})$/', $postcode );
 				break;
 			case 'ES':
+			case 'FI':
```

Comment:
```suggestion
			case 'FI':
			case 'EE':			
```

Estonia has the same postcode format as well, perhaps consider adding it, too?

Reply:
Thanks for the heads up. I'll add it to this PR